pkg/apps/blackduck/latest/containers/environs.go

```diff
@@ -37,4 +43,8 @@ func GetHubKnobs() map[string]string {
 		"DATA_RETENTION_IN_DAYS":            "180",
 		"MAX_TOTAL_SOURCE_SIZE_MB":          "4000",
 	}
+	if versionSAMLSignatureVersionSupportedByDefault, _ := util.IsVersionGreaterThanOrEqualTo(version, 2020, time.February, 0); versionSAMLSignatureVersionSupportedByDefault {
```

Comment:
Only set it if it's exactly 2019.12.1.  For future versions, this is set by default in containers.